manifests/istio-telemetry/kiali/templates/kiali-kialicr.yaml

```diff
@@ -0,0 +1,118 @@
+{{- if .Values.kiali.enabled }}
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kiali
+    release: {{ .Release.Name }}
+annotations:
+  ansible.operator-sdk/verbosity: "3"
+spec:
+{{- if .Values.kiali.fullSpec }}
+{{ toYaml .Values.kiali.fullSpec | indent 2 }}
+{{- else }}
+  ###################################################################
+  # For details about the supported settings,
+  # see https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml"
+  istio_component_namespaces:
+    grafana: "{{ .Values.global.telemetryNamespace }}"
+    pilot: "{{ .Values.global.configNamespace }}"
```

Comment:
Can this be renamed `istiod`, or can pilot and istiod be set leading to the same result?

Reply:
"pilot" setting is necessary specifically so Kiali can have a reasonable default to determine what URL to use when determining what version of Istio is installed in the control plane (i.e. `'http://istio-pilot.' + <this istio_component_namespaces.pilot setting> + ':8080/version`). This setting is used [in the Kiali operator here](https://github.com/kiali/kiali/blob/v1.15.0/operator/roles/default/kiali-deploy/tasks/main.yml#L224).

For backwards compat with Istio 1.4 and prior, this setting will be needed. There is still `pilot` in Istio 1.5 as I understand it (its just rolled into istiod) - so this could still be a valid name. Not sure if we need to explicitly define "istiod" here as Kiali doesn't need it or will use it.